app/components/rdf-input-fields/mandataris-fractie-selector.js

```diff
@@ -68,42 +69,37 @@ export default class MandatarisFractieSelector extends InputFieldComponent {
   }
 
   async loadProvidedValue() {
-    const membershipTriples = triplesForPath(this.storeOptions, false).values;
-    if (!membershipTriples.length) {
+    const fractieUris = triplesForPath(this.storeOptions, false).values;
+    if (!fractieUris.length) {
       return;
     }
-    const membershipUri = membershipTriples[0].value;
+    const fractieUri = fractieUris[0].value;
 
-    const matches = await this.store.query('lidmaatschap', {
-      'filter[:uri:]': membershipUri,
-      include: 'binnen-fractie',
+    const matches = await this.store.query('fractie', {
+      'filter[:uri:]': fractieUri,
     });
-    this.membership = matches.at(0);
+    this.selectedFractie = matches.at(0);
   }
 
+  // TODO Maybe turn into task?
   @action
   async onSelectFractie(fractie) {
     this.updating = true;
-    const newMembership = this.store.createRecord('lidmaatschap', {
-      binnenFractie: fractie,
-    });
-    await newMembership.save();
-    this.membership = newMembership;
+    const uri = fractie.uri;
 
-    // Note: this process creates a new membership whenever the value of the fractie selector changes.
-    // For history purposes, we shouldn't delete the old membership so if we want to view a previous
-    // version of the mandataris forms, we can still see the old membership.
-    // Memberships should therefore also never be modified, only created and deleted.
-    replaceSingleFormValue(this.storeOptions, new NamedNode(newMembership.uri));
+    replaceSingleFormValue(this.storeOptions, uri ? new NamedNode(uri) : null);
+    this.selectedFractie = fractie;
+    // TODO Why is this useful? Also hasn't been declared as a field.
```

Comment:
this is from the inputfield component that we are extending. the forms use it for internal stuff